cooltools/expected.py

```diff
@@ -987,6 +1377,287 @@ def blocksum_asymm(
     )
 
 
+# user-friendly wrapper for diagsum_symm and diagsum_pairwise - part of new "public" API
+def get_cis_expected(
+    clr,
+    view_df=None,
+    symmetric=True,
```

Comment:
make it return either symmetric expected only, or return "everything" cis - e.g. intra- and inter-arm expected.

Reply:
intra_only =True- people like it
intra inter
matching_pairs
...